PlayerControls/sources/AdVideoControls.swift

```diff
@@ -64,25 +67,49 @@ public final class AdVideoControls: UIViewController {
         seekerView.accessibilityLabel = props.seeker?.accessibilityLabel ?? ""
         remainingPlayTimeLabel.text = props.seeker?.remainingPlayTime
         airplayActiveView.isHidden = props.airplayActiveViewHidden
+        
+        adSkipButton.titleLabel?.textAlignment = .center
+        
+        switch props.adSkipState {
+        case .unavailable:
+            adSkipButton.isHidden = true
+        case .available:
+            if adSkipButton.isEnabled == false {
+                adSkipButton.isEnabled = true
+            }
+            adSkipButton.isHidden = false
+            adSkipButton.titleLabel?.text = "Skip"
+        case .awaiting(let time):
+            adSkipButton.isHidden = false
+            adSkipButton.isEnabled = false
+            adSkipButton.titleLabel?.alpha = 0.5
+            adSkipButton.titleLabel?.text = "Skip in \(time)"
```

Comment:
I'm not sure that you can change the text in such way. You have to use `setTitle("", for: )`

Reply:
You can't :)